src/ompl/base/PlannerTerminationCondition.h

```diff
@@ -120,6 +121,32 @@ namespace ompl
 
         /** \brief Return a termination condition that will become true \e duration seconds in the future (wall-time), but is checked in a separate thread, every \e interval seconds; \e interval must be less than \e duration */
         PlannerTerminationCondition timedPlannerTerminationCondition(double duration, double interval);
+
+        /** \brief Return a termination condition that will become true as soon as the problem definition has an exact solution */
+        PlannerTerminationCondition exactSolnPlannerTerminationCondition(ompl::base::ProblemDefinitionPtr pdef);
+
+        /** \brief A class to run a planner for a specific number of iterations. Casts to a PTC for use with Planner::solve */
+        class IterationTerminationCondition
```

Comment:
A couple of concerns:
- I am worried that people will compare different algorithms as a function of number of iterations. Since different algorithms can do wildly different things within an iteration (one alg. can take 1ms per iteration, while another takes 1s), this comparison is not meaningful.
- Why not derive from PlannerTerminationCondition? I guess you don't want it to run in a separate thread?